apps/protailwind/src/pages/index.tsx

```diff
@@ -20,14 +41,17 @@ const Home: NextPage = () => {
     }
   }, [router])
 
+  const {redeemableCoupon, RedeemDialogForCoupon} = useCoupon(couponFromCode)
```

Comment:
currently a "golden ticket" is the only way to purchase so needs the redeem dialog